requirements-dev.in

```diff
@@ -8,4 +8,5 @@ marshmallow-sqlalchemy == 0.23.1 #related to the marshmallow issue (to avoid con
 marshmallow<2.21.1 #schema duplicate issues https://github.com/marshmallow-code/marshmallow-sqlalchemy/issues/121
 nodeenv
 pyyaml>=4.2b1
-sqlalchemy < 1.4.0 # ImportError: cannot import name '_ColumnEntity' https://github.com/sqlalchemy/sqlalchemy/issues/6226
\ No newline at end of file
+sqlalchemy < 1.4.0 # ImportError: cannot import name '_ColumnEntity' https://github.com/sqlalchemy/sqlalchemy/issues/6226
+urllib3<=1.26.16
```

Comment:
Here and elsewhere: can we add a comment on why this pin is necessary similar to other entries?